crt_portal/cts_forms/views.py

```diff
@@ -225,6 +225,16 @@ def setup_filter_parameters(report, querydict):
 
 @login_required
 def index_view(request):
+    profile_form = ProfileForm()
+    # Check for Profile object, then add filter to request
+    if hasattr(request.user, 'profile') and request.user.profile.intake_filters:
+        request.GET = request.GET.copy()
+        request.GET.setlist('assigned_section', request.user.profile.intake_filters.split(','))
```

Comment:
`request.user.profile.intake_filters.split(',')` is used twice, can be pulled out as a variable